### PR TITLE
Major version update confirmation

### PR DESCRIPTION
**Pull request by obfuscode**
## Overview

Adds a password confirmation to the one-click update for major versions.

## Nature of This Change

- [ ] 🐛 Fixes a bug
- [X] 🚀 Implements a new feature
- [ ] 🛁 Refactors existing code
- [ ] 💅 Fixes coding style
- [ ] ✅ Adds tests
- [ ] 👽 Adds new dependency
- [ ] 🔥 Removes unused files / code
- [ ] 🔒 Improves security

## Is this backwards compatible?

- [X] Yes
- [ ] No

### CODE REVIEW

File: system/ee/EllisLab/ExpressionEngine/View/_shared/footer.php [hunk 1]
```diff
@@ -6,20 +6,22 @@
...
-						{
-							if ($vital_update)
-							{
-								$version_class .= ' app-about__version--update-vital';
+						} elseif ($update_available) {
+							$version_major = explode('.', APP_VER, 2)[0];
+							$update_version_major = explode('.', $new_version['version'], 2)[0];
+
+							if (version_compare($version_major, $update_version_major, '<')) {
+								$update_is_major_version = true;
```
Comment (intoeetive) [inline system/ee/EllisLab/ExpressionEngine/View/_shared/footer.php:19]: I think we need to make sure to not apply "vital" class even if update is vital, as that is only relevant per-major-release

Comment (obfuscode) [reply system/ee/EllisLab/ExpressionEngine/View/_shared/footer.php:19]: I've modified this to be an elseif so it won't show the vital update styles if there's already a major update available.

**Comment by ops-andy** [comment]
@obfuscode link for docs: `/installation/updating.html#upgrading-major-versions`

File: system/ee/EllisLab/ExpressionEngine/View/_shared/footer.php (hunk 1 above)
Comment (intoeetive) [reply system/ee/EllisLab/ExpressionEngine/View/_shared/footer.php:19]: I think this is good. Let's package and test it!

**Review by intoeetive — APPROVED** [review]

**Pull request merged**